src/fpm/cmd/new.f90

```diff
@@ -575,6 +575,48 @@ subroutine cmd_new(settings)
     call run('git init ' // settings%name)
 contains
 
+function default_user(what) result(user)
+  implicit none
+  character(len=5), intent(in) :: what
```

Comment:
Fixed length here is a bit dangerous, why not
```suggestion
  character(len=*), intent(in) :: what
```
Also the function name does not really match the actual intent of the procedure, I would suggest something like `get_default_metadata` or similar.